diff --git a/node_modules/xmldom/lib/dom-parser.js b/node_modules/xmldom/lib/dom-parser.js
index bb4dda7..1014c33 100644
--- a/node_modules/xmldom/lib/dom-parser.js
+++ b/node_modules/xmldom/lib/dom-parser.js
@@ -172,7 +172,7 @@ DOMHandler.prototype = {
 	 * @link http://www.saxproject.org/apidoc/org/xml/sax/ErrorHandler.html
 	 */
 	warning:function(error) {
-		console.warn('[xmldom warning]\t'+error,_locator(this.locator));
+		// console.warn('[xmldom warning]\t'+error,_locator(this.locator));
 	},
 	error:function(error) {
 		console.error('[xmldom error]\t'+error,_locator(this.locator));
